docs/en/seatunnel-engine/rest-api-v1.md

```diff
@@ -2,7 +2,12 @@
 sidebar_position: 11
 ---
 
-# RESTful API
+**Notes:**
+
+It is recommended to use the v2 version of the Rest API. The v1 version is deprecated and will be removed in the future.
+
+
+# RESTful API V1
```

Comment:
we should not add notes before first header. It let page title not right.
![image](https://github.com/user-attachments/assets/2da4c2b5-a433-4ef6-9cdf-949760bae760)

Please update to 
```
# RESTful API V1

:::caution warn
It is recommended to use the v2 version of the Rest API. The v1 version is deprecated and will be removed in the future.
:::

```